assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -246,6 +259,11 @@ function MediaPane(props) {
     Flags.INCREMENTAL_SEARCH_DEBOUNCE_MEDIA
   );
 
+  const getSearchCountText = () => {
+    const unit = media.length > 1 ? 'results' : 'result';
+    return `${media.length} ${unit} found`;
+  };
```

Comment:
Please don't do this :-)

First of all, text needs to be translatable.

Second, using plural when length > 1 is only true for _some_ languages. So this works in English, but not in other languages.

Third, string concatenation is strongly discouraged for translatable text.

The correct way to do this:

```
sprintf(
  /* translators: %d: number of results. */
  _n( '%d result found', '%d results found', 'web-stories' ),
  media.length
);
```

Since this is so simple, no need to have a separate `getSearchCountText()` function for this.

Reply:
Got it, will keep in mind.